packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -19,7 +19,7 @@ import {useOverlayPosition} from '@react-aria/overlays';
 import {useTooltipTrigger} from '@react-aria/tooltip';
 import {useTooltipTriggerState} from '@react-stately/tooltip';
 
-const DEFAULT_OFFSET = 7; // Closest visual match to Spectrum's mocks
+const DEFAULT_OFFSET = -1; // Closest visual match to Spectrum's mocks
```

Comment:
why negative?

Reply:
the design feedback wanted a distance of 4px between the button and the tooltip while we had a distance of 12px, hence the 7 - 8 = -1 for the default offset here. Didn't want to modify useOverlayPosition's math since that would affect other overlays so went for this default offset modification since we already had it defined